### ISSUE TITLE
Settings: Some options and buttons don't work

### ISSUE THREAD

**Issue by chr314** [lilyinstarlight/nixos-cosmic#351]
Some buttons and options don't work in settings, like the button to add input sources or to add custom shortcut 

https://github.com/user-attachments/assets/798c8c88-7da2-4638-9611-89e5fb321fc3

**Comment by Tommimon** [comment]
Since is not shown in the video, I will add that also editing existing keyboard shortcuts doesn't work for me

**Comment by Tommimon** [comment]
Today I updated the COSMIC flake and now almost all the buttons you show in your video work. I can edit keyboard shortcuts and add custom ones, I can add and remove input sources.

In the 3 dots menu of input sources, `Settings` and `View keyboard leyout` still don't work, maybe it is an upstream issue?

**Comment by lilyinstarlight** [comment]
> In the 3 dots menu of input sources, `Settings` and `View keyboard leyout` still don't work, maybe it is an upstream issue?

Yeah some stuff still needs to be updated for keyboard layouts. See <https://github.com/lilyinstarlight/nixos-cosmic/issues/74#issuecomment-2353273156>
